Modules/Phasing.lua

```diff
@@ -666,15 +666,15 @@ function Phasing.IsSpawnVisible(phase)
     end
 
     if phase == phases.RHEA_NEW_KARGATH then
-        return complete[27888] and (not complete[27897]) and (not questLog[27897]) or false
+        return complete[27888] and ((not complete[27897]) and ((not questLog[27897]) or (questLog[27897] and questLog[27897].isComplete == 0))) or false
     end
 
     if phase == phases.RHEA_DRAGONS_MOUTH then
-        return complete[27769] and (questLog[27826] and questLog[27826].isComplete == 1) and ((not complete[27832]) and ((not questLog[27832]) or (questLog[27832] and questLog[27832].isComplete == 0))) or false
+        return complete[27794] and ((not complete[27832]) and ((not questLog[27832]) or (questLog[27832] and questLog[27832].isComplete == 0))) or false
     end
 
     if phase == phases.RHEA_HIDDEN_CLUTCH then
-        return complete[27832] or questLog[27832] or complete[27897] or questLog[27897] or false
+        return ((complete[27832] or (questLog[27832] and questLog[27832].isComplete == 1)) or complete[27897] or (questLog[27897] and questLog[27897].isComplete == 1)) or false
```

Comment:
27832 also does not have any objectives so `questLog[27832].isComplete == 1` will always be `true`. And the same goes for 27897.